ports/esp32/boards/OLIMEX_ESP32_POE/board.json

```diff
@@ -0,0 +1,24 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "BLE",
+        "WiFi",
+        "MicroSD",
+        "Battery Charging",
+        "Ethernet",
+        "PoE",
+        "Breadboard friendly"
+    ],
+    "id": "esp32",
```

Comment:
@robert-hh this line should not be here, it's making it link to the generic esp32 firmware

Reply:
Is the `id` tag required?